### PR TITLE
Resolves header component

### PR DESCRIPTION
**Pull request by monotera**
Footer is finished with some animations. I had some issues with positioning in a specific width, therefore, I created a new variable. I didn't put the logo because I thought it look better just the name, if you want I can change it.

### CODE REVIEW

File: src/components/Burger.vue [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<template></template>
+
+<script>
+export default {};
+</script>
+
+<style></style>
```
Comment (Manuel-2011) [inline src/components/Burger.vue]: This was a file for testing, right? if it is, please delete it.

File: src/components/Header.vue [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+            :class="{ active: active_el == 2 }"
+            href="#clases-component"
+            >Clases</a
+          >
+        </li>
+        <li>
+          <a
+            @click="activate(3)"
+            :class="{ active: active_el == 3 }"
+            href="#galeria-component"
```
Comment (Manuel-2011) [inline src/components/Header.vue]: I used "gallery-component" as the div id for the gallery feature, are we going to use the name conventions in english or in spanish?

**Review by Manuel-2011 — COMMENTED** [review]
I am amazed by the transition effects you made, good job. I wonder if you could make the mobile view menu sticky so the menu is always available for the user.

File: src/components/Burger.vue (hunk 1 above)
Comment (monotera) [reply src/components/Burger.vue]: Thanks, I had forgotten about this

File: src/components/Header.vue (hunk 2 above)
Comment (monotera) [reply src/components/Header.vue]: In English, I just didn't want to leave it blank, I was waiting to see what you called it to change it.

**Comment by monotera** [comment]
I tried but couldn't make it sticky, I don't know why. I always used the sticky position. So in the end what I did was close the menu if someone was scrolling so the menu wouldn't look weird and I was planning to implement a button with an arrow to go to the top, like the web page that brian gave us. but I did not know whether to do it on this ticket or create a new one, if you want I can do it on this one, I don't think it will delay me

File: src/components/Header.vue (hunk 2 above)
Comment (monotera) [reply src/components/Header.vue]: I have change it

Comment (Manuel-2011) [reply src/components/Header.vue]: Great.

**Review by Manuel-2011 — CHANGES_REQUESTED** [review]
I tried to resolve the issue of the sticky menu and I was able to solve it. You just have to change the position property of the header to "fixed" and you can add width 100% and that's all.

**Review by Manuel-2011 — APPROVED** [review]

**Pull request merged**